core/src/main/python/wlsdeploy/tool/validate/validator.py

```diff
@@ -128,7 +129,20 @@ def validate_in_standalone_mode(self, model_dict, variable_map, archive_file_nam
         self._logger.entering(archive_file_name, class_name=_class_name, method_name=_method_name)
         self.__validate_model_file(cloned_model_dict, variable_map, archive_file_name)
 
-        self._logger.exiting(class_name=_class_name, method_name=_method_name)
+        summary_handler = SummaryHandler.findInstance()
+        if summary_handler is not None:
+            summary_level = summary_handler.getMaximumMessageLevel()
+            if summary_level == Level.SEVERE:
+                status = Validator.ValidationStatus.INVALID
+            elif summary_level == Level.WARNING:
+                status = Validator.ValidationStatus.WARNINGS_INVALID
+
+        status = Validator.ValidationStatus.VALID
```

Comment:
Should status be set to VALID **before** checking handler?
Here it will always make the return code PROCEED.